lib/foreman_maintain/concerns/disk_utility.rb

```diff
@@ -0,0 +1,41 @@
+require 'foreman_maintain/concerns/disk_utility/device'
+require 'foreman_maintain/concerns/disk_utility/disk_type'
+require 'foreman_maintain/concerns/disk_utility/disk_io'
+require 'foreman_maintain/concerns/disk_utility/file_io'
+
+module ForemanMaintain
+  module Concerns
+    module DiskUtility
+      include Logger
+      MIN_SPEED_MBPS = 80
+      DEFAULT_DIRS = ['/var/lib/pulp', '/var/lib/mongodb', '/var/lib/pgsql'].freeze
+
+      def self.included(klass)
+        klass.extend(self)
+      end
+
+      def pass?
```

Comment:
This looks good. For unit testing, you should be able to test the check similar way as any other object. Something like `check = Checks::DiskSpeedMinimal.new; assert check.pass`